sycl/test-e2e/AddressSanitizer/common/demangle-kernel-name.cpp

```diff
@@ -0,0 +1,20 @@
+// REQUIRES: linux, cpu, aspect-fp64
```

Comment:
Is there a reason we use `double` for this test? Seems to me like we could just use `int` and avoid the additional requirement of `aspect-fp64`.

Reply:
Good point.